py/selenium/webdriver/support/expected_conditions.py

```diff
@@ -259,11 +260,14 @@ class invisibility_of_element_located(object):
     locator used to find the element
     """
     def __init__(self, locator):
-        self.locator = locator
+        self.target = locator
 
     def __call__(self, driver):
         try:
-            return _element_if_visible(_find_element(driver, self.locator), False)
+            target = self.target
+            if not isinstance(self.locator, WebElement):
```

Comment:
This is causing the tests to fail with `AttributeError: 'invisibility_of_element' object has no attribute 'locator'`